dev/acf-export.json

```diff
@@ -722,6 +704,13 @@
                     "value": "page"
                 }
             ],
+            [
+                {
+                    "param": "post_type",
```

Comment:
The page header field group shouldn't be present for single posts (this is a custom change for this theme)